migration/20190515-reformat-existing-geographic-data.py

```diff
@@ -0,0 +1,31 @@
+from nose.tools import set_trace
+from sqlalchemy.sql import *
+import json
+import os
+import sys
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..")
+sys.path.append(os.path.abspath(package_dir))
+from api.admin.geographic_validator import GeographicValidator
+from core.model import (
+    Library,
+    production_session
+)
+from core.model.configuration import ConfigurationSetting
+
+_db = production_session()
+
+area_settings = _db.query(ConfigurationSetting).filter(
+    or_(ConfigurationSetting.key == "service_area", ConfigurationSetting.key == "focus_area")).filter(
+    or_(ConfigurationSetting._value != None, ConfigurationSetting._value == "")
+    ).all()
+
+settings_with_wrong_format = [s for s in area_settings if isinstance(json.loads(s._value), (list,))]
```

Comment:
I think this will work if the bad value is a JSON list like `["Waterford, CT"]`, but the value I saw in a real circ manager was `Waterford, CT`. Trying to load that string as JSON will cause a `ValueError`.

So maybe try to load the setting as JSON, and treat it as needing repair if it raises a `ValueError` or the result isn't a dictionary.